build_tools/github/check_build_trigger.sh

```diff
@@ -4,9 +4,10 @@ set -e
 set -x
 
 COMMIT_MSG=$(git log --no-merges -1 --oneline)
+BRANCH_NAME=$(echo "${GITHUB_REF##*/}")
```

Comment:
`GITHUB_REF` is something like `refs/heads/feature-branch-1`. Filtering to get the branch name.